styleguide/components/11-base/fonts/_icons.scss

```diff
@@ -0,0 +1,141 @@
+@font-face {
```

Comment:
Don't like the 4 space tabs in here :P but thats a personal opinion :-D
If this file is generated automatically we should ignore it in git.

Reply:
Should be solved.